src/scripts/rechunk.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+import dask.array as da
+import argparse
+import time
+
+
+def timeit(method):
+    def timed(*args, **kw):
+        ts = time.time()
+        result = method(*args, **kw)
+        te = time.time()
+        if 'log_time' in kw:
+            name = kw.get('log_name', method.__name__.upper())
+            kw['log_time'][name] = int((te - ts) * 1000)
+        else:
+            print('%r  %2.2f ms' %
+                  (method.__name__, (te - ts) * 1000))
+        return result
+    return timed
+
+
+@timeit
+def load(source_array):
+    return da.from_zarr(source_array)
+
+
+@timeit
+def resize(arr, chunks):
+    return arr.rechunk(chunks)
+
+
+@timeit
+def convert(resized, target_array):
+    da.to_zarr(resized, target_array)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--dimensions", type=int, default="5", metavar="DIMS",
+                        help="number of chunks to parse (%(default)s)")
+    parser.add_argument("--distributed", action="store_true",
+                        help="enable distributed dashboard (%(default)s)")
+    parser.add_argument("source_array",
+                        help="array to copy from. must exist.")
+    parser.add_argument("target_array",
+                        help="array to write to. may not exist.")
```

Comment:
Seems it better not or I get,
> ValueError: path '' contains an array

Reply:
Is this if the target already already exists?

Reply:
It is, yes.

Reply:
So the exception is bad, but matches the help text, right?

Reply:
No, "must not exist" would match the exception?

Reply:
That's not how I parse it, but ok. I'll update.

Reply:
Oh, haha, I thought it was saying I didn't have to pre-create it! Thank you.